docs/guide/overview-of-event-sources.md

```diff
@@ -131,6 +131,15 @@ functions:
     events:
       - sns: dispatch
 ```
+Or if you have a pre-existing topic ARN, you can just provide the topic ARN instead:
```

Comment:
ok done. However I don't see why we're duplicating docs in multiple places! The overview is meant to give a quick start (ie. I don't see the extra options for APIG in the overview) and to ignore the details. This is an overall docs issue though :)